### PR TITLE
Feat: about-us page

### LINKED ISSUE #7: Impress / About page
About us + legal

### PR DESCRIPTION
**Pull request by dorianim**
This adds a new page at [`/about-us`](https://feat-about-us-page.speedclimbing.pages.dev/about-us) with some information about the team.

- [ ] some information about team members
  - [X] Dorian
  - [ ] Jakob
  - [X] Peter
- [x] contributors
- [x] contact form
- [x] imprint
- [x] privacy policy

Fixes #7

### CODE REVIEW

File: src/compnonents/shared/AnchorPoint.svelte [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+	export let id: string;
+</script>
+
+<a class="none anchor" {id} href="/">#</a>
```
Comment (lamorak13) [inline src/compnonents/shared/AnchorPoint.svelte:5]: remove none class

File: src/compnonents/shared/AnchorPoint.svelte [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+<style>
+	a.anchor {
```
Comment (lamorak13) [inline src/compnonents/shared/AnchorPoint.svelte:8]: Use tailwind classes for better consistency

**Review by lamorak13 — CHANGES_REQUESTED** [review]

**Comment by lamorak13** [comment]
I really like the design and the idea to also display contributors. The only thing I would consider in terms of design is to make alignment consistent in the "who we are section".

File: src/compnonents/shared/AnchorPoint.svelte (hunk 2 above)
Comment (dorianim) [reply src/compnonents/shared/AnchorPoint.svelte]: You are completely right!

**Review by lamorak13 — APPROVED** [review]
Like it!

**Pull request merged**